core/src/kvs/tx.rs

```diff
@@ -47,6 +47,8 @@ use std::ops::Range;
 use std::sync::Arc;
 use uuid::Uuid;
 
+const LQ_CAPACITY: usize = 100;
```

Comment:
Would it make sense to make that configurable?

Reply:
Yes, we can make it configurable in a separate PR :) Creating a ticket on linear for that now thanks